demos/python_demos/image_retrieval_demo/models.lst

```diff
@@ -0,0 +1,2 @@
+# This file can be used with the --list option of the model downloader.
+image-retrieval-????
```

Comment:
Does it make sense to run the demo with face/person ReID models?

Reply:
no, it does not